Source/File.System.Stub.IntegrationTests/File.System.Stub.IntegrationTests.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-		<TargetFramework>netstandard2.0</TargetFramework>
-	<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TargetFramework>netcoreapp2.1</TargetFramework>

```

Comment:
Rozumiem że testy muszą chodzić w .net core?

Reply:
Tak, .NET Standard to nie jest "środowisko", to tylko standard, więc trzeba było wybrać .NET Core albo .NET Framework, a że Travis nie ogarnia .NET Framework, to padło na .NET Core.